packages/wdio-spec-reporter/src/index.js

```diff
@@ -90,6 +91,17 @@ class SpecReporter extends WDIOReporter {
         this.write(`${divider}\n${prefacedOutput.join('\n')}\n`)
     }
 
+    /**
+     * get link to saucelabs job
+     */
+    getTestLink ({ config, sessionId }) {
+        if (config.hostname.includes('saucelabs')) {
```

Comment:
What do you think about making this more generic and checking if options/config has a property for the link, and then making the services populate the link? This way we could easily allow browserstack, testingbot, etc to plugin?